src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingPreview.jsx

```diff
@@ -40,99 +41,131 @@ import LabelsDesign from '../../../plots-and-tables/components/LabelsDesign';
 const { Panel } = Collapse;
 
 const EmbeddingPreview = () => {
```

Comment:
split this into two components depending on plot type: make one component for `embeddingCategorical ` and one for `embeddingContinuous`. Make the `EmbeddingPreview` call them depending on what is needed. Iva will make a ticket for later refactoring these new components to be used as generic components

Reply:
Refactored into `CategoricalEmbeddingPlot.jsx` and `ContinuousEmbeddingPlot.jsx`